changelog/fragments/bundle-pm-supported-kinds.yaml

```diff
@@ -0,0 +1,3 @@
+entries:
+  - description: When generating bundles and packagemanifests, add all resources supported by OLM.
+    kind: change
```

Comment:
I almost want to say this is a net-new feature because we explicitly do this for RBAC and no other types, but I am ok with "change" since the explicitness isn't user-facing.